vllm/v1/engine/utils.py

```diff
@@ -73,6 +73,7 @@ class EngineHandshakeMetadata:
 
     addresses: EngineZmqAddresses
     parallel_config: dict[str, Union[int, str, list[int]]]
+    config_hash: Optional[str] = None
```

Comment:
suggest rename for clarity
```suggestion
    parallel_config_hash: Optional[str] = None
```